FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder.php

```diff
@@ -283,4 +279,10 @@ public function addUserGroup($usergroup)
     {
         $this->parameters['usergrouphash'] = StaticHelper::calculateUsergroupHash($this->shopKey, $usergroup);
     }
+
+
+    public function addFlag(string $key, $value)
```

Comment:
Interesting. Shouldn't this fail for PHP 5.6 because the string type hint is only supported in PHP 7.x?

Reply:
There are a lot of typehints all over the place, i don't know